03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/sh
```

Comment:
Wrong path to shell. Usial location is /bin/sh

Reply:
Done.

Reply:
This script uses some bash features which are not supported by sh. So the line should be #!/bin/bash

Reply:
Done.